src/middlewares/validationHandler/guestValidator.js

```diff
@@ -0,0 +1,25 @@
+import { check, validationResult } from "express-validator";
+
+exports.guestValidator = [
+    check('name').trim().not().isEmpty()
+    .withMessage("Please provide your Name")
+    .isLength({min: 5, max: 50})
+    .withMessage("Name must be within 5 to 50 Characters"),
+    check('email').normalizeEmail().isEmail()
+    .withMessage("Please provide a valid Email"),
+    check('phone').trim().not().isEmpty()
+    .withMessage("Please provide your Phone Number")
+    .isLength({min: 11, max: 11})
+    .withMessage("Please provide a valid Phone Number"),
+    check('gender').not().isEmpty()
+    .withMessage("Please Select your Gender")
+    
+]
+
+exports.guestValidation = (req, res, next) => {
```

Comment:
Let this function take an argument, i.e the array you validating.
So in the route all you have to do is call this function and pass the array into for it to validate it.

Reply:
Okay. I have done that.